package.json

```diff
@@ -2,16 +2,28 @@
   "name": "maker-tools",
   "version": "0.0.2",
   "description": "Tools to help market makers on Hashflow operate on the exchange",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/",
+    "LICENSE",
+    "README.md"
+  ],
   "scripts": {
+    "build:tsc": "rm -rf dist/ && tsc",
+    "build:lint": "eslint .",
+    "build": "yarn build:lint && yarn build:tsc",
```

Comment:
CircleCI config should be updated to reflect this? 

Why do we have `build:lint` and `eslint` and also `build:tsc` and `tsc`? Are those redundant, or is there a reason why we have both?

Reply:
I brought over these rules from market-maker-js. Perhaps @gxmxni-hashflow can comment on the reasoning?

Reply:
Per DM with @gxmxni-hashflow, removing redundant targets.